k8s/namespaces/rd/rd-judicial-api-integration/rd-judicial-api-integration.yaml

```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/judicial-api:prod-5f536da-20210827120457
+      image: hmctspublic.azurecr.io/rd/judicial-api:prod-5f536da-20210827120457   #{"$imagepolicy": "flux-system:rd-judicial-api-integration"}
```

Comment:
Can you change this policy to `rd-judicial-ref-api`? It looks to be using the same, I would remove the image repo/policy for `rd-judicial-ref-api-integration`

Reply:
Image policy/repo now updated.